src/mosquito/queue.cr

```diff
@@ -1,11 +1,13 @@
 module Mosquito
   class Queue
     ID_PREFIX = {"mosquito"}
+    QUEUES    = %w(waiting scheduled pending dead config)
 
-    WAITING = "queue"
-    PENDING = "pending"
-    SCHEDULED = "scheduled"
-    DEAD = "dead"
+    {% for q in QUEUES %}
+      def {{q.id}}_q
+        redis_key {{q}}, name
+      end
+    {% end %}
```

Comment:
This is a nice refactor, thanks. 

The name "waiting" was a weird choice because "queue_q" was a bad name.